KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift

```diff
@@ -233,10 +237,29 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
         headerView.anchor(leading: baseStackView.leadingAnchor, trailing: baseStackView.trailingAnchor, paddingLeft: titleInset, paddingRight: titleInset)
         headerView.heightAnchor.constraint(greaterThanOrEqualToConstant: 90.0).isActive = true
         
-        cancellationInfoLabel.anchor(top: headerView.bottomAnchor, leading: baseStackView.leadingAnchor, trailing: baseStackView.trailingAnchor, paddingTop: 20.0, paddingLeft: 20.0, paddingBottom: 20.0, paddingRight: 20.0)
+        cancellationInfoLabel.anchor(top: headerView.bottomAnchor,
+                                     leading: baseStackView.leadingAnchor,
+                                     trailing: baseStackView.trailingAnchor,
+                                     paddingTop: 20.0,
```

Comment:
magic numbers... can you define a constant for this value ( similar to what we have at line 22)

Reply:
done